.github/actions/build-vsix/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Build VSIX
+description: Builds the Visual Studio extension package for the solution.
+
+inputs:
+  version-number:
+    description: >
+      If provided, will be used directly as the version of the VSIX. Otherwise, the version will be automatically
+      incremented.
+    required: false
+    default: ''
```

Comment:
The action/workflow files in the PR have a mixed usage of `'` and `"`, let's stick to `"`.